src/extras/TextureTransform.js

```diff
@@ -0,0 +1,115 @@
+import { default as Matrix } from '../core/math/Matrix';
+
+const tempMat = new Matrix();
+
+/**
+ * class controls uv transform and frame clamp for texture
+ */
+export default class TextureTransform {
```

Comment:
Hey @englercj  this is code not required by the core. It's only useful for TilingSprite at the moment. That being the case I believe it should be there.